std_msgs/Float32MultiArray in_embedding
std_msgs/Int16 in_label
std_msgs/String in_name
---
std_msgs/Int16 out_label